### ISSUE TITLE
Fix docs

### ISSUE THREAD

**Issue by WellingtonCosta** [WellingtonCosta/convalida#36]

**Comment by emanuelet** [comment]
+1. At the moment is a bit unclear how to implement the new v3 version

**Comment by WellingtonCosta** [comment]
Hi, @emanuelet 

I'm rewriting the website. As soon as possible it will be available. Maybe this helps: https://github.com/WellingtonCosta/convalida/tree/master/sample/src/main/java/convalida/sample

**Comment by emanuelet** [comment]
Maybe I should a separate issue but I try to update I get this error of the compiler

**Comment by WellingtonCosta** [comment]
I couldn't understand exactly what happened. Is your project using AndroidX artifacts and the jetifier?

**Comment by emanuelet** [comment]
no actually I still have the support libraries v27.1.1

**Comment by WellingtonCosta** [comment]
You need to use AndroidX artifacts and Jetifier to use Convalida 3.x.y.

**Comment by emanuelet** [comment]
fair enough, it's probably worthwhile to mention in the installation instructions. Thanks for the help
